Seeing thousands of these in logs "error canceling retrieval"

### Checklist

- [X] This is **not** a question or a support request. If you have any boost related questions, please ask in the [discussion forum](https://github.com/filecoin-project/boost/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/boost/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/boost/issues) and the [discussion forum](https://github.com/filecoin-project/boost/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/boost/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to boost.

### Boost component

- [X] boost daemon - storage providers
- [ ] boost client
- [ ] boost UI
- [ ] boost data-transfer
- [ ] boost index-provider
- [ ] Other

### Boost Version

```text
boostd --version
boostd version 1.6.0+git.9c0b02c

lotus-miner version
Daemon:  1.20.3+mainnet+git.b332db0a3+api1.5.0
Local: lotus-miner version 1.20.3+mainnet+git.b332db0a3
```


### Describe the Bug

I have been seeing these ERROR level messages in my Boostd logs since 1.5.2

Boost node is idling (no deals), seeing a lot of autoretrievals and lotus-miner events in lotus-miner log as follows:
> 2023-03-10T22:04:51.616Z	INFO	stores	paths/http_handler.go:188	SERVE Alloc check /remote/unsealed/s-t01278-4529/8/allocated/0/34091302912
2023-03-10T22:05:06.854Z	INFO	stores	paths/http_handler.go:188	SERVE Alloc check /remote/unsealed/s-t01278-4206/8/allocated/0/34091302912

I should not be seeing so many events at a warn level (thousands every few seconds)

### Logging Information

```text
2023-03-10T22:00:31.620Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089744", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.620Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.620Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089732", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.620Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089725", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1678396438396332461", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1678396438396332460", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1678396438396332459", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640128182", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640128181", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089703", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089674", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640128158", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1678381793047682618", "err": "No channel for channel ID 12D3KooWHPSfSxqzhwSSXbooq4kNmFyg9ATfRm6Bnb92TGRWbUVN--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089667", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089658", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.621Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089655", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089513", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089445", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640128038", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089436", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089397", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640127999", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "0", "err": "No channel for channel ID 12D3KooWLrL2jkS8hE7mJxnKsckq4omUvyenHezSRcCQCTB7JjTx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1675382075798089050", "err": "No channel for channel ID 12D3KooWNTiHg8eQsTRx8XV7TiJbq3379EgwG6Mo3V3MdwAfThsx--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640127838", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640127836", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.622Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640127834", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
2023-03-10T22:00:31.623Z        ERROR   rtrvlog rtvllog/retrieval_log.go:261    error canceling retrieval       {"dealID": "1677058887640127833", "err": "No channel for channel ID 12D3KooWPmm3iECH2vjdezfxwKeEbHJ5RmcyT3Bz2zsPZmNZURiS--0"}
```


### Repo Steps

1. Run 'tail -f /market/logs/boost.log'
2. See error 'error canceling retrieval'
...

--- a/retrievalmarket/rtvllog/retrieval_log.go
+++ b/retrievalmarket/rtvllog/retrieval_log.go
@@ -258,7 +258,7 @@ func (r *RetrievalLog) gcRetrievals(ctx context.Context) {
 				chid := datatransfer.ChannelID{Initiator: row.PeerID, Responder: row.LocalPeerID, ID: row.TransferID}
 				err := r.dataTransfer.CloseDataTransferChannel(ctx, chid)
 				if err != nil {
-					log.Errorw("error canceling retrieval", "dealID", row.DealID, "err", err)
+					log.Debugw("error canceling retrieval", "dealID", row.DealID, "err", err)
 				} else {
 					log.Infof("Canceled retrieval %s, older than %s", row.DealID, r.stalledTimeout)
 				}

